### DISCUSSION TITLE
setup problem using bootstrap-vue

### DISCUSSION THREAD

**Discussion by ollixx** [TotallyInformation/node-red-contrib-uibuilder#179] [Q&A] [answered]
Hi Julian,
I am trying out the latest uibuilder version.
What I did:
- I placed a uibuilder node and gave it an URL
- I added vue, bootstrap and bootstrap-vue in the library manager
- I opened the site and see this error:
```
GET https://xxx/uibuilder/vendor/vue/dist/vue.min.js net::ERR_ABORTED 404 (Not Found)
config.js:8 Uncaught TypeError: Cannot read properties of undefined (reading 'prototype')
    at config.js:8:22
    at bootstrap-vue.min.js:10:225
    at bootstrap-vue.min.js:10:234
(anonym) @ config.js:8
(anonym) @ bootstrap-vue.min.js:10
(anonym) @ bootstrap-vue.min.js:10
index.js:24 Uncaught ReferenceError: Vue is not defined
    at index.js:24:13
```

Did I oversee something, or maybe there are old templates in the project?

Would love to get some help here, Cheers!
:oliver

**Comment by ollixx** [comment, marked as answer]
OK, found it myself.
- latest vue version delivers `vue.global.js` and not `vue.min.js`.
I still get errors, but that's probably, because vue 3 is newer than the templates...

**Comment by ollixx** [comment]
OK. found more :-)
boostrap-vue does NOT work on vue 3.x yet.
So my problem was to install vue (3.x) before installing bootstrap-vue. 
I installed vue@2.6.0 and now everything is like expected.

**Comment by TotallyInformation** [comment]
Hi, thanks for this. Sorry I didn't get to responding quicker. Yes, VueJS is a bit of a problem at the moment since they moved their default deployment from v2 to v3 when most dependent projects don't yet support v3 and migration from v2 to v3 isn't very easy.

Thankfully, uibuilder v5 allows you to install dependent packages using any version/branch/etc so installing vue as `vue@2` will ensure that you get the correct version.
